packages/kotti-ui/source/next/kotti-field-date/hooks.ts

```diff
@@ -161,6 +163,20 @@ const usePickerDimensionsFix = <DATA_TYPE extends Values>({
 			if (isPickerVisible(dateComponent)) {
 				dateComponent.picker.$el.style.width = popperWidth
 				dateComponent.picker.$el.style.height = popperHeight
+				/**
+				 * when we open the date-picker we trigger `showPicker`
+				 * https://github.com/ElemeFE/element/blob/649670c55a45c7343eb7148565e2d873bc3d52dd/packages/date-picker/src/picker.vue#L806
+				 * which calls `updatePopper` from the mixin vue-popper
+				 * https://github.com/ElemeFE/element/blob/649670c55a45c7343eb7148565e2d873bc3d52dd/src/utils/vue-popper.js#L120
+				 * which calls the prototype-defined function `update` from popper.js
+				 * https://github.com/ElemeFE/element/blob/3ceec7aa6a7ab46e61dbafa4c68b77ba09384b40/src/utils/popper.js#L224
+				 *
+				 * HACK: to force re-compute the boundaries of the popper element (picker) (in `update` function)
+				 * after we set its width and height, since this is computed in the constructor of the popper, and
+				 * at the instance of initialization, we don't have the new dimensions set, thus resulting in
+				 * a placement issue.
+				 */
```

Comment:
I'd say it'd be good to add a link to the JIRA ticket or (even better, maybe?) a description of how to reproduce the placement issue

Otherwise, future us might find this and have no clue if the bug was already fixed somewhere else

Reply:
it's reproducible on the docs though, regardless of the JIRA issue or do you want that to see a video? bc I can also upload that on this issue

Reply:
No, my point is that right now we know what issue this is, but in the future we or someone else might want to refactor this and they'd want to make sure they're not accidentally re-introducing the bug

So, a comoment that basically contains the requirements & symptoms of the issue (or, alternatively — but harder — a unit test) would be amazing for someone in that situation

Right now, I sorta know what this is about, but that won't be the case in the future